fpdf/fpdf.py

```diff
@@ -1974,7 +1974,15 @@ def _out(self, s):
         if(self.state == 2):
             self.pages[self.page]["content"] += (s + "\n")
         else:
-            self.buffer += (s + "\n")
+            if PY3K:
+                if not isinstance(s, bytes):
+                    s = s.encode('latin1')
+                s += b"\n"
```

Comment:
If I am right about s always being str at this point, change it to

```python
s += "\n"
if(self.state == 2):
    self.pages[self.page]["content"] += s
else:
    if PY3K:
        s = s.encode('latin1')
    self.stream.write(s)
```

Otherwise, maybe you could use the b() function from the py3k module.